### PR TITLE
Add Installation Guide with Ansible

### LINKED ISSUE #142: Ansible Role for conmon
I am now maintenance an Ansible Role for conmon installation (https://github.com/alvistack/ansible-role-conmon), by downloading the static binary release archive and extract it for following OS:
- Ubuntu 16.04/18.04/19.10
- CentOS/RHEL 7/8
- openSUSE 15.1

Could we also introduce this as an alternative installation method, e.g. under https://github.com/containers/conmon/blob/master/install.md#conmon-installation-instructions?

### PR DESCRIPTION
**Pull request by hswong3i**
Provide additional installation options by using Ansible with https://github.com/alvistack/ansible-role-conmon from source code.

Fixes #142

### CODE REVIEW

**Comment by haircommander** [comment]
LGTM, thanks @hswong3i

File: install.md [hunk 1]
```diff
@@ -37,13 +37,28 @@ cd conmon
 make
 ```
 
+## Install with Ansible
+
+An [Ansible Role](https://github.com/alvistack/ansible-role-conmon) is also available for automate above steps:
```
Comment (TomSweeneyRedHat) [inline install.md:42]: nit:  "for automate" -> "to automate the"

**Comment by TomSweeneyRedHat** [comment]
one nit, otherwise LGTM

File: install.md (hunk 1 above)
Comment (hswong3i) [reply install.md]: fixed ;-)

**Comment by TomSweeneyRedHat** [comment]
LGTM and happy green test buttons.

File: install.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ However, if you'd like to try building it from source, follow the steps below.
 
 ## Fedora, CentOS, RHEL, and related distributions:
 
-```bash
+``` bash
```
Comment (saschagrunert) [inline install.md:8]: I don't hink we need those additional spaces, right?

File: install.md [hunk 3]
```diff
@@ -37,13 +37,28 @@ cd conmon
...
 There are three options for installation, depending on your environment. Each can have the PREFIX overridden. The PREFIX defaults to `/usr/local` for most Linux distributions.
 
-- `make install` installs to `$PREFIX/bin`, for adding conmon to the path.
-- `make podman` installs to `$PREFIX/libexec/podman`, which is used to override the conmon version that Podman uses.
-- `make crio` installs to `$PREFIX/libexec/crio`, which is used to override the conmon version that CRI-O uses.
+  - `make install` installs to `$PREFIX/bin`, for adding conmon to the path.
+  - `make podman` installs to `$PREFIX/libexec/podman`, which is used to override the conmon version that Podman uses.
+  - `make crio` installs to `$PREFIX/libexec/crio`, which is used to override the conmon version that CRI-O uses.
 
+<!-- end list -->
```
Comment (saschagrunert) [inline install.md:62]: The same applies for those and the comment.

Comment (hswong3i) [reply install.md:62]: It is done by ‘pandoc -t gfm -f gfm --wrap preserve -o install.md install.md’

File: install.md (hunk 2 above)
Comment (hswong3i) [reply install.md]: rebase done

File: install.md (hunk 3 above)
Comment (hswong3i) [reply install.md]: rebase done

**Comment by saschagrunert** [comment]
It looks like that we use the CRI-O master for testing. I'll see what's wrong there with the integration test.

**Comment by haircommander** [comment]
docs only, manually merging

**Pull request merged**